blog/_src/posts/2017-05-24-plc-russia.md

```diff
@@ -0,0 +1,39 @@
+    Title: Programming Language Conference in Russia
+    Date: 2017-05-24T12:25:17
+    Tags: by Artem Pelenitsyn
+
+In April 3--5 I took part into a Russian conference exclusively devoted to programming languages: [Programming Languages and Compilers][PLC] ([Google.Translated version of the site][PLCGT]). I was a member of organizing committee and had a paper there. 
+
+This is the first conference in Russia highly focused on our area of PL. At least for the last several decades (I believe, there were conferences of the kind back in USSR). The conference was devoted to the memory of prominent Soviet PL-researcher from Rostov-on-Don, Adolf Fuksman, who really forestalled aspect-oriented programming paradigm.
```

Comment:
I'll replace *forestalled* with *pioneered* -- I wasn't sure about the wording here. Though *pioneered* sounds a bit over passionated. But I can do this.

I agree that `more` after second par. is even better.